src/workflow/ArcanistLandWorkflow.php

```diff
@@ -254,6 +246,15 @@ public function getArguments() {
           'git',
         ),
       ),
+      'skip-submitqueue-checks' => array(
+        'help' => pht(
+          'Skip SubmitQueue checks and jump to the front of the queue. '.
+          'This is meant to be used for emergency cases when the queue is blocked/backlogged. '.
+          'BREAKGLASS is required.'),
```

Comment:
Do we also want to add message that it requires Charter policies to be added for the use in SQ or should we that for the call error message to reflect it?